granulate_utils/linux/cgroups_client/base_cgroup.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import os
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+from granulate_utils.linux.cgroups_client.exceptions import (
+    MissingCgroup,
+    MissingController,
+    SkippedCgroup,
+    UnsupportedCgroup,
+)
+
+PID_CGROUPS = Path("/proc/self/cgroup")
+CGROUP_PARENT_PATH = Path("/sys/fs/cgroup")
+
+IGNORE_LIST = ["kubepods", "docker", "ecs"]
+
+
+class HIERARCHIES(Enum):
+    memory = "memory"
+    cpu = "cpu,cpuacct"
```

Comment:
This enum seems not very much used (the values at least), do you have a particular use for it ?

Reply:
https://github.com/Granulate/granulate-utils/pull/56/files#diff-85381afe3dc85f5e26b1f381a942c019c2b7830deaf508bc0e35805deb5a9a7bR63